content/doc/tutorials/build-a-multibranch-pipeline-project.adoc

```diff
@@ -545,4 +550,4 @@ Because Jenkins is extremely extensible, it can be modified and configured to ha
 To learn more about what Jenkins can do, check out:
 
 * The link:/doc/tutorials[Tutorials overview] page for other introductory tutorials.
-* The link:/doc/book[User Handbook] for more detailed information about using Jenkins, such as link:/doc/book/pipeline[Pipelines] (in particular link:/doc/book/pipeline/syntax[Pipeline syntax]) and the link:/doc/book/b
\ No newline at end of file
+* The link:/doc/book[User Handbook] for more detailed information about using Jenkins, such as link:/doc/book/pipeline[Pipelines] (in particular link:/doc/book/pipeline/syntax[Pipeline syntax]) and the link:/doc/book/b
```

Comment:
```suggestion
* The link:/doc/book[User Handbook] for more detailed information about using Jenkins, such as link:/doc/book/pipeline[Pipelines] (in particular link:/doc/book/pipeline/syntax[Pipeline syntax]).
```
Removing the last bit of this text as it is the beginning of creating a link, but the rest of the URL is missing. If there's a good choice to update this link to, then it can be added.